src/test/java/org/apache/accumulo/proxy/ProxyServerTest.java

```diff
@@ -69,14 +68,14 @@ public void updateAndFlushClosesWriterOnExceptionFromAddCells() throws Exception
   @Test
   public void updateAndFlushClosesWriterOnExceptionFromFlush() throws Exception {
     ProxyServer server = EasyMock.createMockBuilder(ProxyServer.class)
-        .addMockedMethod("getWriter", ByteBuffer.class, String.class, WriterOptions.class)
+        .addMockedMethod("getWriter", String.class, String.class, WriterOptions.class)
         .addMockedMethod("addCellsToWriter", Map.class, BatchWriterPlusProblem.class).createMock();
     BatchWriter writer = EasyMock.createMock(BatchWriter.class);
     BatchWriterPlusProblem bwpe = new BatchWriterPlusProblem();
     bwpe.writer = writer;
     MutationsRejectedException mre = EasyMock.createMock(MutationsRejectedException.class);
 
-    final ByteBuffer login = ByteBuffer.wrap("my_login".getBytes(UTF_8));
+    final String login = "my_login";
```

Comment:
The change to use a shared proxy secret instead of the previous login mechanism is okay, but it left a bunch of tests with confusing variable names, which make the tests hard to follow.